extensions/swagger-ui/deployment/src/test/java/io/quarkus/swaggerui/deployment/NoConfigTest.java

```diff
@@ -0,0 +1,26 @@
+package io.quarkus.swaggerui.deployment;
+
+import static org.hamcrest.Matchers.equalTo;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusUnitTest;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+
+public class NoConfigTest {
+
+    @RegisterExtension
+    static final QuarkusUnitTest config = new QuarkusUnitTest()
+            .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class));
+
+    @Test
+    public void shouldUseDefaultConfig() {
+        RestAssured.when().get("/swagger-ui").then().statusCode(200);
+        RestAssured.when().get("/swagger-ui/index.html").then().statusCode(200);
```

Comment:
One additional comment: I wonder if we should test if the pattern `/openapi` is in the page returned. IIRC, it would be something like `.body(containsString("/openapi"))`.

Reply:
Done
https://github.com/quarkusio/quarkus/pull/1573/files#diff-28ec27bba9df0f01afcda9af942bdc63R22